tests/unit/table_scrambler_test.php

```diff
@@ -178,7 +178,13 @@ public function test_it_scrambles_names($except, $expected) {
     }
 
```

Comment:
I think if you add the php comment
```php
    /**
     * @expectedException invalid_parameter_exception
     */
```

it may work for both versions. I remember getting annoyed with the different phpunit versions and doing something like that.